vendor/bat-native-ads/src/bat/ads/internal/search_providers.cc

```diff
@@ -47,4 +49,42 @@ bool SearchProviders::IsSearchEngine(
   return is_a_search;
 }
 
+std::string SearchProviders::ExtractSearchQueryKeywords(
+    const std::string& url) {
+  GURL visited_url = GURL(url);
+  if (!visited_url.is_valid()) {
+    return "";
```

Comment:
nit: breaks copy elision as should declare `search_query_keywords` and return empty query